js/okex.js

```diff
@@ -3731,6 +3731,45 @@ module.exports = class okex extends Exchange {
         return await this.modifyMarginHelper (symbol, amount, 'add', params);
     }
 
+    async fetchMaxTradeAmount (symbol, marginMode, code = undefined, params = {}) {
```

Comment:
I'm not sure if we should add a new method for this, we have fetchTradingLimit / fetchTradingLimits for that matter.

Reply:
yeah i'd agree, we shouldn't try to add new methods unless we can be sure to unify them

Reply:
> I'm not sure if we should add a new method for this, we have fetchTradingLimit / fetchTradingLimits for that matter.

Okex's `limit.amount` is also set to undefined for max, the type in `limit.amount.max` is a different kind of max though so setting it to the max you receive here wouldn't really be right. It looks like `limit.amount.max` is the max that a user can trade without considering how much is in their wallet, while the max in this method is the max considering their wallet balance